src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -257,6 +267,16 @@
     </{{ element|default('div') }}>
 {%- endblock form_row %}
 
+{% block file_row -%}
+    <{{ element|default('div') }} class="form-group">
```

Comment:
In this case the element can't be another thing than a `div` element, maybe it's better to directly write `div` in place of use the `element` var ? It works like it, it's just a question/suggestion. ;-)